### PR TITLE
fix(core): fix slider handle edge position

### PR DESCRIPTION
**Pull request by Smiranin**
#### Please provide a link to the associated issue.
Closes SAP/fundamental-ngx#5529

#### Please provide a brief summary of this pull request.
Added wrapper for slider handle to prevent getting out of the strip edges of slider track. (look related issue)
Also fixed position of "before element" of handle for rtl.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by dimamarksman** [comment]
I think this fix should be added in the `fundamental-styles` itself. 
@InnaAtanasova what do you think?

**Comment by dimamarksman** [comment]
I'm not even sure that it's a slider's problem at all. 
Slider's consumer should take into account that a container, that contains the slider, shouldn't use `overflow hidden`

**Comment by InnaAtanasova** [comment]
> I'm not sure where the fix should be done, here or in the `fundamental-styles` itself.
> @InnaAtanasova what do you think?

This should def be handled in fundamental-styles and then tested in fund-ngx.
@Smiranin please open a PR in Fundamental-styles first.

**Review by InnaAtanasova — APPROVED** [review]

**Comment by dimamarksman** [comment]
Looks good,
the only minor thing that focus outline is cut

image

**Review by dimamarksman — APPROVED** [review]
Looks good,
the only minor thing that focus outline is cut

image

**Comment by InnaAtanasova** [comment]
Please check the failing tests

**Review by mikerodonnell89 — APPROVED** [review]

File: libs/core/src/lib/slider/slider.component.ts [hunk 1]
```diff
@@ -61,6 +61,11 @@ export class SliderComponent implements OnInit, OnChanges, OnDestroy, ControlVal
     @HostBinding('attr.id')
     id = 'fd-slider-id-' + sliderId++;
 
+    /** Whether to apply cozy mode. */
+    @Input()
+    @HostBinding('class.fd-slider--lg')
+    cozy: boolean = null;
```
Comment (Betrozov) [inline libs/core/src/lib/slider/slider.component.ts:67]:
```suggestion
    cozy = false;
```

**Review by Betrozov — CHANGES_REQUESTED** [review]

**Review by Betrozov — APPROVED** [review]

**Pull request merged**